rustlang/install.sh

```diff
@@ -2,9 +2,35 @@
 set -e
 set -u
 
+# shellcheck disable=SC2016
+
 function __install_rust() {
+
+    mkdir -p "$HOME/.cargo/bin"
```

Comment:
Move these lines 9 and 10 (mkdir and pathman) below the rust installer and that should be good enough.

We shouldn't need to edit .cargo/env.